src/frontend/components/Felleskomponenter/Feilside/FeilsideInnhold.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react';
+
+import { BodyShort, Heading, Link, List } from '@navikt/ds-react';
+
+import { useSpråk } from '../../../context/SpråkContext';
+import { LocaleType } from '../../../typer/common';
+
+import { GåTilBakeTilForrigeSideLenke } from './GåTilbakeTilForrigeSideLenke';
+import { LastInnSidenPåNyttLenke } from './LastInnSidenPåNyttLenke';
+
+export const FeilsideInnhold: FC = () => {
```

Comment:
Når man skal støtte flere spårk er det som regel vanlig å ta å bruk f.eks. `react-intl`, da kan man gi hver tekst en egen id og slå opp teksten via denne id'en, slik trenger man ikke å skrive egen HTML for hvert språk. 

se https://www.npmjs.com/package/react-intl